TitanRobotics2024/src/main/java/frc/robot/Auto/Actions/TurnDegreesAction.java

```diff
@@ -0,0 +1,97 @@
+package frc.robot.Auto.Actions;
+
+import frc.robot.Subsystem.DriveBase;
+import frc.robot.Subsystem.Gyro;
+import frc.robot.Subsystem.PositionEstimation;
+
+import javax.swing.text.Position;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+
+public class TurnDegreesAction implements Actions 
+{
+    Timer timer;
+    private double currentDegrees = 0;
+    private double desiredDegrees;
+    private double targetDegrees;
+    private double turn;
+    private double toleranceDegrees = 0.0025;
```

Comment:
Same for moving this (See line 30)

Reply:
no particular reason for the relation, just was copied over